packages/frontend/src/components/txConfirm/ConfirmSend.tsx

```diff
@@ -58,27 +66,37 @@ const ConfirmSend = (props: Props) => {
   return (
     <div className={styles.root}>
       <div className={styles.title}>
-        <div
-          style={{
-            marginBottom: '1rem',
-          }}
-        >
-          <Typography variant="h6" color="textSecondary">
-            Send{' '}
-            <strong>
-              {commafy(source.amount, 5)} {source.token.symbol}
-            </strong>{' '}
-            from {source.network.name} to {dest?.network?.name}
-          </Typography>
-        </div>
-        <div>
-          <Typography variant="subtitle2" color="textSecondary">
-            Estimated Received:
-          </Typography>
-          <Typography variant="subtitle2" color="textPrimary">
-            {estimatedReceived}
-          </Typography>
-        </div>
+        <Typography variant="h6" color="textSecondary">
+          Send{' '}
+          <strong>
+            {commafy(source.amount, 5)} {source.token.symbol}
```

Comment:
suggestion: include token and chain symbols to improve ux

![DeepinScreenshot_select-area_20230926092057](https://github.com/hop-protocol/hop/assets/168240/70048a0c-6d58-4643-9386-11dfdb9b6880)